src/main/modules/session/index.ts

```diff
@@ -0,0 +1,36 @@
+import config from 'config';
+import RedisStore from 'connect-redis';
+import type { Express } from 'express';
+import session from 'express-session';
+import { Redis } from 'ioredis';
+
+export class Session {
+  enableFor(app: Express): void {
+    const sessionMiddleware: session.SessionOptions = {
+      secret: 'PCS-SECRET', //TODO: replace this
+      resave: false,
+      saveUninitialized: true,
+      cookie: { sameSite: 'strict' },
+      name: config.get('session.cookieName'),
+    };
+
+    const redis = new Redis(config.get('secrets.pcs.redis-connection-string'));
+    console.log('conn str = ', config.get('secrets.pcs.redis-connection-string'), process.env.REDIS_CLOUD_URL);
```

Comment:
Remove this or put a condition as this logs the secret across all envs?

Reply:
was only for debugging purposes and wouldnt be commited.